tests/accelerators/test_accelerator_connector.py

```diff
@@ -454,7 +455,7 @@ def test_plugin_accelerator_choice(accelerator: Optional[str], plugin: str):
     assert isinstance(trainer.strategy, DDPShardedStrategy)
 
     with pytest.deprecated_call(match="Passing .* `strategy` to the `plugins`"):
-        trainer = Trainer(plugins=plugin, num_processes=2)
+        trainer = Trainer(plugins=plugin, accelerator="cpu", num_processes=2)
```

Comment:
what about this?

Reply:
Wow, yeah, I’m gonna change that

Reply:
Wait, that might be one of the reverted tests. *testing*

Reply:
yeah, that's what I meant :))